src/lib/wallet/MultipleHttpProvider.js

```diff
@@ -8,7 +8,7 @@ import logger from '../logger/js-logger'
 const { providers } = Web3
 const { HttpProvider } = providers
 const log = logger.child({ from: 'MultipleHttpProvider' })
-const connectionErrorRe = /connection (error|timeout)|invalid json rpc/i
+const connectionErrorRe = /connection (error|timeout)|invalid json rpc|Invalid JSON RPC/i
```

Comment:
no sense of this change because of `/i` modifier already applied

```suggestion
const connectionErrorRe = /connection (error|timeout)|invalid json rpc/i
```

Reply:
ah right, got it.
also got replaced by your last commits